packit_service/worker/testing_farm.py

```diff
@@ -66,12 +66,21 @@ def _payload(self, build_id: int, chroot: str) -> dict:
 
         """
         compose, arch = self.get_compose_arch(chroot)
+        # FIXME FIXME FIXME !!!
+        # This tells TF to load fmf metadata from main/master of the base project.
+        # What we need is to load it from self.metadata.commit_sha of the FORKED project.
+        # This is ugly hack, but I need to make this work at least like this ASAP
+        # and find proper way later. Sorry.
+        url = self.metadata.project_url
+        ref = self.project.default_branch
+        # url = ? url of the fork from which the PR has been created ?
+        # ref = self.metadata.commit_sha
```

Comment:
Yes, the fork seems like a way to go. I think we already have that in `self.metadata`.

Reply:
I was of course already looking into `self.metadata` but  give it another try if you say so.
Merging this for now. Thanks